### ISSUE TITLE
Error: Failed to save the clsk...-sigKeyIndex item to the file system

### ISSUE THREAD

**Issue by jondubois** [Capitalisk/ldpos-commander#57]
A user reported this issue on Ubuntu:

```
Error: Failed to save the clskc5143b850634eac0b84b6bd14613758e5b16fda8-sigKeyIndex item to the file system because of error: EACCES: permission denied, open '/usr/lib/node_modules/ldpos-commander/node_modules/ldpos-client/data/clskc5143b850634eac0b84b6bd14613758e5b16fda8-sigKeyIndex'
Error: Type help to see all available commands.
```

**Comment by jondubois** [comment]
It should specify a specific directory to save the key index files or else it will try to save them inside the module by default which can be a problem if the module is installed globally.

This path can be specified using `storeDirPath` in the `LDPoSClient` constructor.

```js
new LDPoSClient({ storeDirPath: '/path/to/key/directory/' })
```

**Comment by jondubois** [comment]
This appears to have been fixed already but it wasn't published to npm.

**Comment by jondubois** [comment]
I've published it in `v1.1.2`.
